stories/bar/bar.stories.js

```diff
@@ -9,30 +9,30 @@ export default {
         description: `The bar component is a container that holds titles, buttons and input controls. The contents are distributed into three areas: left, middle and right. This component’s primary function is to display page **headers** and **footers**.
```

Comment:
What does "input control" mean exactly? Aren't the buttons considered to be input controls? Would it be "buttons 'with' input controls"? @SAP/fundamental-denoland-team